### ISSUE TITLE
Why use sigmoid instead of softmax in segmentation network??

### ISSUE THREAD

**Issue by consolexinhun** [leohsuofnthu/Pytorch-IterativeFCN#6]
Why use sigmoid instead of softmax in segmentation network?

i think this is multi-class segmentation problem......

**Comment by rw1995** [comment]
论文中有原话
![image](https://user-images.githubusercontent.com/37976371/141227595-77d68340-9ad4-47eb-8d72-eb2155626b53.png)
